src/components/Redemptions/RedemptionsPage.tsx

```diff
@@ -119,20 +155,20 @@ class RedemptionsPage extends React.Component<IProps, null> {
   }
 
   private renderProposalsPerDAO(): RenderOutput[] {
-    const { currentAccountAddress, data: [daos, proposals] } = this.props;
+    const { currentAccountAddress, data: proposals } = this.props;
 
     const arc = getArc();
 
     const daosPerId = new Map<Address, IDAOState>();
```

Comment:
more meaningful name might be:  `daoStatePerDaoAddress`.  Is a bit confusing that the id is the address.

Reply:
indeed it is, blame the client library

Reply:
`daosPerId` is right here, not in the client